react/Badge/Readme.md

```diff
@@ -1,55 +1,96 @@
-`<Badge />` component wraps the element on which you want to apply a badge. In the following examples, it is applied to compact `<ButtonAction />`.
+`<Badge />` component wraps the element you want to apply a badge to. It is based on the [Badge component from Material-UI](https://v3.material-ui.com/api/badge/).
 
-You should use `<Badge />` to provide additional information related to the wrapped content.
+### Basic Badges
 
-This component spreads all other props to its root element.
+The default version of badges is used to indicate an amount of new things that the user should see. This could be a number of new messages in an inbox or the number of new files in a folder.
 
-#### Available types
+The `dot` variant can be used to indicate that the item it is applied to _itself_ is new. A `dot` badge on a folder for example would indicate that the folder is new, not that it contains new elements.
 
-Default type is `normal`
+```
+import MuiCozyTheme from 'cozy-ui/transpiled/react/MuiCozyTheme';
+import Badge from 'cozy-ui/transpiled/react/Badge';
+import Icon from 'cozy-ui/transpiled/react/Icon';
+import Variants from 'cozy-ui/docs/components/Variants';
+
+const initialVariants = [
+  { error: false, dot: false, large: false, small: false },
+  { error: true, dot: true, large: false, small: false },
+  { error: false, dot: false, large: false, small: false },
+];
 
+
+<MuiCozyTheme>
+  <Variants initialVariants={initialVariants}>{
+    variant => (
+      <p>
+        <Badge badgeContent={4} color={variant.error ? 'error' : variant.secondaryColor ? 'secondary' : 'primary'} variant={variant.dot ? 'dot' : 'standard'} size={variant.large ? 'large' : variant.small ? 'small' : 'medium'} anchorOrigin={{vertical: variant.bottom ? 'bottom' : 'top', 'horizontal': variant.left ? 'left' : 'right'}}>
+          <Icon icon="circle-filled" size={variant.large ? '32' : variant.small ? '16' : '24'} color="var(--slateGrey)" />
+        </Badge>
+      </p>
+    )
+  }</Variants>
+</MuiCozyTheme>
 ```
-import Badge from 'cozy-ui/transpiled/react/Badge'
-import ButtonAction from 'cozy-ui/transpiled/react/ButtonAction';
 
-<div>
-  <p>
-    <Badge content="2" type="normal">
-      <ButtonAction label='Normal' rightIcon='openwith' compact />
-    </Badge>
-  </p>
+### Qualifier badge
```

Comment:
I wonder if variant="info" would not be better than variant="qualifier". I find "qualifier" a bit odd to be honest.

Also, I wonder if we should not have an InfoBadge component instead of choosing a variant of the badge component.